### PR TITLE
#167729771 setting up database

### PR DESCRIPTION
**Pull request by chibytez**
#### What does this PR do?
adding database
#### Description of Task to be completed?
setting up database
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
#167729771
#### Screenshots (if appropriate)
Capture

sgg

#### Questions:

### CODE REVIEW

File: src/database/config.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  },
+  test: {
+    url: process.env.TEST_DATABASE_URL,
+    dialect: 'postgres',
+  },
+  production: {
+    url: process.env.DATABASE_URL,
+    dialect: 'postgres',
+  },
+}
```
[the remark this answers is not in the document]
Comment (mthamayor) [reply src/database/config.js]: Please kindly resolve this. @chibytez

File: src/database/config.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require('dotenv').config()
```
[the remark this answers is not in the document]
Comment (mthamayor) [reply src/database/config.js]: Please kindly resolve this. @chibytez

**Review by Stylll — CHANGES_REQUESTED** [review]
Hey @chibytez, can you add an `.env.sample` file containing samples of the environment variables needed to use the database

File: .env.sample [hunk 3]
```diff
@@ -0,0 +1,3 @@
+DATABASE_URL=
+DEV_DATABASE_URL=postgres://kuvsqczu:g_zoAu9DRRF3kINngz1sT8UAe-d26Dyv@raja.db.elephantsql.com:5432/kuvsqczu
```
Comment (Stylll) [inline .env.sample]: @chibytez This is fine. I hope the values here are not the actual values for development

**Review by Stylll — CHANGES_REQUESTED** [review]
@chibytez Please resolve the merge conflicts

**Review by mthamayor — CHANGES_REQUESTED** [review]
You haven't handled the conflicts successfully. Please let the team know if you need help with that.
Please stop using es5 features and start using ES6 since limiting is already present.

**Review by davidfrank96 — APPROVED** [review]
nice compartmentalization with the Db connection strigs

**Review by okafor-chidimma — APPROVED** [review]
I have reviewed...Excellent work @chibytez  but I would suggest you remove or update the screenshot that you have attached to your PR before you close it

File: README.md [hunk 4]
```diff
@@ -1,6 +1,20 @@
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (Stylll) [inline README.md]: hey @chibytez , Please fix the merge conflict issue here.

**Review by Stylll — CHANGES_REQUESTED** [review]
Hey @chibytez Please fix the merge conflict issue in the Readme.md file.

**Review by okabamac — APPROVED** [review]
Good work sir

File: README.md (hunk 4 above)
Comment (chibytez) [reply README.md]: have done that ,thanks

**Review by Stylll — APPROVED** [review]

**Pull request merged**